### ISSUE TITLE
Ebpf metrics (are cilium or falco behaving?)

### ISSUE THREAD

**Issue by teemow** [giantswarm/roadmap#1548]

**Comment by teemow** [comment]
@alex-dabija @stone-z @cornelius-keller @gawertm your teams are currently dealing with tooling that relies on ebpf. Would it make sense for one of you to take this into your teams as a story?

Adding @giantswarm/team-atlas for awareness of this too.

**Comment by alex-dabija** [comment]
Yes, it makes sense for one of the KaaS teams (Hydra or Rocket) to own this, but I don't know which team has more capacity at the moment.
